AdobeStockAsset/Test/_files/category_rollback.php

```diff
@@ -0,0 +1,19 @@
+<?php
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+declare(strict_types=1);
+
+use Magento\TestFramework\Helper\Bootstrap;
+use Magento\AdobeStockAssetApi\Api\CategoryRepositoryInterface;
+
+$objectManager = Bootstrap::getObjectManager();
+
+try {
+    /** @var CategoryRepositoryInterface $categoryRepository */
+    $categoryRepository = $objectManager->get(CategoryRepositoryInterface::class);
+    $categoryRepository->deleteById(42);
+} catch (\Exception $exception) {
+
```

Comment:
I don't think it's a good idea to suppress exceptions in the tests

Reply:
@sivaschenko I used your commit https://github.com/magento/magento2/pull/27499/files#diff-00c291aa1c720f56ddc146c36da5b66fR18 for the rollback as an example. Will remove